conf/machine/include/rpi-default-providers.inc

```diff
@@ -7,6 +7,8 @@ PREFERRED_PROVIDER_virtual/libgles2 ?= "${@bb.utils.contains("MACHINE_FEATURES",
 PREFERRED_PROVIDER_virtual/libgl ?= "${@bb.utils.contains("MACHINE_FEATURES", "vc4graphics", "mesa", "mesa-gl", d)}"
 PREFERRED_PROVIDER_virtual/mesa ?= "${@bb.utils.contains("MACHINE_FEATURES", "vc4graphics", "mesa", "mesa-gl", d)}"
 PREFERRED_PROVIDER_virtual/libgbm ?= "${@bb.utils.contains("MACHINE_FEATURES", "vc4graphics", "mesa", "mesa-gl", d)}"
+PREFERRED_PROVIDER_virtual/vlc ?= "vlc"
+PREFERRED_PROVIDER_virtual/ffmpeg ?= "ffmpeg"
```

Comment:
Wouldn't we want to default these to the rpi-provided ones? It sounds like a sane default configuration for our machines.

Reply:
Haha, yea making change